example/objective_c/README.md

```diff
@@ -0,0 +1,36 @@
+# Objective C example
+
+This example shows how to use ffigen to generate bindings for an Objective C
+library. It uses the AVFAudio framework to play audio files.
+
+```
+dart play_audio.dart test.mp3
+```
+
+## Config notes
+
+The ffigen config for an Objective C library looks very similar to a C library.
+The most important difference is that you must set `language: objc`. If you want
+to filter which interfaces are included you can use the `objc-interfaces:`

```

Comment:
`objc-interfaces:`? The top level readme.md specifies `objcInterfaces`. We should fix the inconsistency.